aziotctl/aziotctl-common/src/lib.rs

```diff
@@ -2,11 +2,31 @@
 
 #![deny(rust_2018_idioms)]
 #![warn(clippy::all, clippy::pedantic)]
+#![allow(
+    clippy::default_trait_access,
+    clippy::let_unit_value,
+    clippy::module_name_repetitions,
+    clippy::similar_names,
+    clippy::too_many_lines,
+    clippy::type_complexity
+)]
 
 use std::collections::BTreeMap;
 
 use serde::{Deserialize, Serialize};
 
+mod restart;
+mod set_log_level;
+mod status;
+mod system_logs;
+
+pub use restart::restart;
+pub use set_log_level::set_log_level;
+pub use status::get_status;
+pub use system_logs::get_system_logs;
+
+pub use log::Level as LogLevel;
```

Comment:
ehhhh, I'd avoid introducing this typedef, since it's re-exporting a foreign type under a different name. 

If you want to "hide" the fact that the underlying type of `LogLevel` is a `log::Level`, then you could introduce a `struct LogLevel(log::Level)` type, but it would add a bunch of boilerplate and is probably overkill.